kubefed/installing-kubefed.adoc

```diff
@@ -0,0 +1,21 @@
+[id="installing-kubefed"]
+= Installing OpenShift KubeFed
```

Comment:
I think we probably want a different name here, but I'm not sure what.

Reply:
@listey @nee1esh can we please decide on the naming once and for all. Afaik the agreement is to use the upstream name, so is this just "KubeFed" or "OpenShift KubeFed"? Not a blocker for this PR but we need to clear this up before release.

Reply:
Thought this was closed, as per Mike B's instruction we use the upstream name, so just KubeFed (I'll check about the OpenShift prefix when he comes online later).

Reply:
@listey any update on the OpenShift prefix? I'd imagine if ours is a productized release we need something to distinguish it as a RH product; won't it get confusing if it's just KubeFed and there's a community operator or something too with the same name?

Reply:
Leaving this as is for now until we have confirmation - we can update it before release. I'm using a variable for the name so it shouldn't be a big fix and shouldn't block this PR for now.